test/e2e/ps_test.go

```diff
@@ -306,7 +306,27 @@ var _ = Describe("Podman ps", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 
 		Expect(session.OutputToString()).To(ContainSubstring(podid))
+	})
+
+	It("podman --pod with a non-empty pod name", func() {
```

Comment:
I am not sure to understand why the new test I added here is failing on:
1. `Cirrus CI / special_testing_rootless TEST_REMOTE_CLIENT:true`
2. `Cirrus CI / testing TEST_REMOTE_CLIENT:true image:fedora-30-libpod-5642998972416000`
3. `Cirrus CI / testing TEST_REMOTE_CLIENT:true image:ubuntu-18-libpod-5642998972416000`

It seems OK for others (hopefully).

On `special_testing_rootless`, it seems to fail for some tests that may not be impacted by the changes I made in this PR (but I am new to this project so I may have missed something):
```
[+1418s] [Fail] Podman start [It] podman failed to start with --rm should delete the container 
[+1418s] /var/tmp/go/src/github.com/containers/libpod/test/e2e/start_test.go:130
[+1418s] 
[+1418s] [Fail] Podman start [It] podman failed to start with --rm should delete the container 
[+1418s] /var/tmp/go/src/github.com/containers/libpod/test/e2e/start_test.go:130
```

Any idea?

Reply:
It seems to be linked to `TEST_REMOTE_CLIENT:true` but unsure what it means.

Reply:
I went with the `SkipIfRemote()` solution proposed here https://github.com/containers/libpod/pull/4748#issuecomment-569119162